File: PHP-ProyectUberClass/car.php
<?php
require_once('account.php');
class car{
    public $id;
    public $driver;
    public $passenger;

    public function __construct($license, $driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar()
    {
        echo "Licencia: $this->license Driver: ".$this->driver;
    }
}
?>

File: Java-ProyectUberClass/car.java
class car{
    Integer id;
    String license;
    account driver;
    Integer passenger;

    public car(String license, account driver){
        this.license= license;
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("License : " + license + "Driver: " + driver.name);
    }
}